src/main/java/uk/gov/hmcts/reform/draftstore/data/DraftStoreDAO.java

```diff
@@ -60,14 +61,24 @@ public SaveStatus insertOrUpdate(String userId, String type, String newDocument)
 
     public void insert(String userId, CreateDraft newDraft) {
         jdbcTemplate.update(
-            "INSERT INTO draft_document (user_id, document, document_type) VALUES (:userId, cast(:doc AS JSON), :type)",
+            "INSERT INTO draft_document (user_id, document, document_type) VALUES (:userId, :doc::JSON, :type)",
             new MapSqlParameterSource()
                 .addValue("userId", userId)
                 .addValue("doc", newDraft.document.toString())
                 .addValue("type", newDraft.type)
         );
     }
 
+    public void update(int id, String userId, UpdateDraft draft) {
```

Comment:
Looks like your plan is to use `userId` in the update query, so that no data is changed when the update is requested by the wrong person. That bit is missing.

Reply:
removed unused param